packages/core/bin/run

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env node
 
-if (process.env.LD_PRELOAD !== 'libjemalloc.so') {
-    process.env.LD_PRELOAD = 'libjemalloc.so'
-    let exitStatus = 0
-    try {
-        require('child_process').execSync(process.argv.join(' '), { stdio: [0, 1, 2] })
-    } catch (error) {
-        exitStatus = error.status
+const child_process = require('child_process')
```

Comment:
This whole block of new code should be wrapped into `try/catch` to not prevent core from running if any of it fails.

Reply:
917d5d4